website/docs/core-concepts/stacks/templating.md

```diff
@@ -138,18 +192,44 @@ To be able to use the `env` function from both templating engines, you can do on
 
 <br/>
 
-### Configuring templating in Atmos stack manifests
+### Configuring Templating in Atmos Stack Manifests
+
+Templating in Atmos can also be configured in the `settings.templates.settings` section in stack manifests.
 
 The `settings.templates.settings` section can be defined globally per organization, tenant, account, or per component.
 Atmos deep-merges the configurations from all scopes into the final result using [inheritance](/core-concepts/components/inheritance).
 
-For example, define [Gomplate Datasources](https://docs.gomplate.ca/datasources/) for the entire organization in the
-`stacks/orgs/acme/_defaults.yaml` stack manifest:
+The schema is the same as `templates.settings` in the `atmos.yaml` [CLI config file](/cli/configuration),
+except the following settings are not supported in the `settings.templates.settings` section:
+
+- `settings.templates.settings.enabled`
+- `settings.templates.settings.sprig.enabled`
+- `settings.templates.settings.gomplate.enabled`
+- `settings.templates.settings.evaluations`
+- `settings.templates.settings.delimiters`
+
+These settings are not supported for the following reasons:
+
+- You can't disable templating in the stack manifests which are being processed by Atmos as `Go` templates 
+
+- If you define the `delimiters` in the `settings.templates.settings` section in stack manifests, 
+  the `Go` templating engine will think that the delimiters specify the beginning and the end of template strings, will 
+  try to evaluate it, which will result in an error
+
+As an example, let's define templating configuration for the entire organization in the `stacks/orgs/acme/_defaults.yaml` 
+stack manifest:
 
 ```yaml title="stacks/orgs/acme/_defaults.yaml"
 settings:
   templates:
     settings:
```

Comment:
I find this super confusing (`settings.templates.settings`), and thought it was a typo when I saw it below